src/backend/cassandra/SettingsProvider.cpp

```diff
@@ -24,6 +24,7 @@
 
 #include <boost/json.hpp>
 
+#include <fstream>
```

Comment:
why we need this header here ?

Reply:
needed for `std::ifstream` down the line. before it was included thru some other headers which got removed i believe